vendor/random/BUILD.gn

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+config("external_config") {
+  include_dirs = [ "." ]
```

Comment:
this will make `#include "random.h"` work which we don't want. I think at least for now we should use `../..` which will make it `#include "vendor/random/random.h"` (`#include "brave/vendor/random/random.h"` also works because `src` is in include_dirs, but we don't necessarily want `brave` in front of everything in brave-core)

Reply:
fixed (with `vendor/brave_random` for now) using `include_dirs = [ "../.." ]` instead